packages/forms/src/UIForm/merge.js

```diff
@@ -1,252 +1,10 @@
-import React from 'react';
```

Comment:
just to make sure; we don't need this anymore even if you use the deprecated version?

Reply:
Yes, this one has never been used and is full of bugs (the main idea was automigrate from old schema to new schema);

Reply:
On our side we don't use it. But now I found `wrapCustomWidget` being used here: https://github.com/Talend/ui/pull/2082/files#diff-0ff6e5979591c52a9139e5f1cff133cdR9 (packages/forms/src/Form.js → packages/forms/src/deprecated/Form.js)

Reply:
Looks like this is fixed now